deployment/centos-package-x64/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM centos:7
 ARG SOURCE_DIR=/jellyfin
 ARG PLATFORM_DIR=/jellyfin/deployment/centos-package-x64
 ARG ARTIFACT_DIR=/dist
-ARG SDK_VERSION=2.2
+ARG SDK_VERSION=3.0
```

Comment:
This is not going to be included in 10.4.1?..

Reply:
Josh is totally switching this when backporting to master.

Reply:
Yep, I'm going to manually correct this bits when I backport it. It avoids a second PR to master immediately after to make them work and also made sure testing was OK.